src/Microsoft.Health.Fhir.CodeGen/CompareTool/CrossVersionMapCollection.cs

```diff
@@ -353,6 +353,20 @@ public static void ProcessCrossVersionFml(string name, FhirStructureMap fml, Dic
         string groupSourceVar = string.Empty;
         string groupTargetVar = string.Empty;
 
+        // Skip (re-)processing some known recursive points
+        if (sourcePrefix == "QuestionnaireResponse.item.item" || targetPrefix == "QuestionnaireResponse.item.item"
```

Comment:
If we need to worry about re-processing, we should track where we have been instead of manually here (since I can see some missing ones).